index.html

```diff
@@ -82,37 +88,36 @@ <h2 id="modalTitle">Add New Item</h2>
     </div>
     </div>
 
+    <!-- FILTERS AND OPTIONS SIDEBAR -->
+     <div id="optionsModal" class="optionsModal" role="dialog" aria-labelledby="optionsModalTitle" aria-modal="true" >
+       <div class="optionsSettingsContainer">
+        <div class="optionsModalTitleContainer">
+        <h2 class="optionsModalTitle">Settings</h2>
+        <button id="closeOptionsModal" class="close" aria-label="Close Settings Panel">&times;</button>
+        </div>
+       <div class="filterInputs">
+        <label class="itemsControlInput">
+          <input type="checkbox" id="autoSort" class="checkbox" />
+          Auto Sort
+          </label>
+          <label class="itemsControlInput">
+            <input type="checkbox" id="hideChecked" class="checkbox" />
+          Hide Checked
+          </label>
+       </div>
+
+       <div class="optionsModalFooter">
+       <button id="backupData" class="downloadDataButton">Download</button>
+       </div>
+       </div>
+     </div>
+
     <!-- SHOPPING ITEMS LIST -->
     <div id="itemsDiv"></div>
     <!-- TOTAL PRICE OF LIST -->
     <div class="stickyPriceFooter">
       <!-- ITEMS CONTROLS -->
       <div id="itemsControl" class="itemsControl">
-        <div class="itemsControlButtons">
-          <!-- TODO: Reset and Delete buttons -->
-          <!-- <button id="resetChecked">Reset Checked</button>
-          <button id="deleteChecked">Delete Checked</button> -->
-        </div>
-        <div class="filters">
-          <label class="itemsControlInput">
-            Auto Sort
-            <input type="checkbox" id="autoSort" class="checkbox" />
-          </label>
-          <label class="itemsControlInput">
-            Hide Checked
-            <input type="checkbox" id="hideChecked" class="checkbox" />
-          </label>
-          <!-- TODO: Sort by Section -->
-          <!-- <label>
-            Sort by Section
-            <input type="checkbox" id="sectionSort" class="checkbox" />
-          </label> -->
-        </div>
-      </div>
-      <div id="showItemsControl" class="showItemsControl">
-        <p id="toggleMessage">Hide Filters</p>
```

Comment:
All of this has been removed - having a piece of ui slide up from the bottom was going to quickly get cumbersome with more options for filtering. And it didn't look very nice with a 'download' button either. 

The footer remains sticky to always see the estimated price, but this might be something we offer with a checkbox in the future as well - I can definitely see a user not wanting to record prices, and simply have a nice shopping list.